test/passing/ite-fit_or_vertical.ml.ref

```diff
@@ -149,3 +149,13 @@ let foo =
   else if Poly.(assoc_of_prec prec_ast = which_child && which_child <> Non)
   then
     foo
+
+let _ =
+  if fooo then ( 
```

Comment:
these ones are not too great. what was the situation for this code in 0.15.0?

Reply:
I observe the same results with 0.15.0